backend/api/src/change-user-info.ts

```diff
@@ -129,26 +128,6 @@ export const changeUser = async (
   }
   log(`Updated ${betRows.length} bets.`)
 
-  log('Updating denormalized user data on contract metrics docs...')
-  const contractMetricsRows = await pg.manyOrNone(
```

Comment:
afaict this is still necessary, why remove it?

Reply:
because it's annoying to re-write to do the same thing, it should just be a join, and the cache moving out of date doesn't impact much. as long as we do the user join / user display cache stuff quickly enough it after this, it should be ok